tests/unit/test_advanced.py

```diff
@@ -57,11 +57,16 @@ def initial_feedback(cls):
         return cls.expected_configuration()["initial_feedback"]
 
     def test_get_configuration(self):
-        self.assertEqual(self.expected_configuration(), self.block.get_configuration())
+        try:
+            self.assertEqual(self.expected_configuration(), self.block.get_configuration())
+        except AssertionError:
+            print self.expected_configuration()
+            print self.block.get_configuration()
+            raise
```

Comment:
This is very useful while debugging a problem, but I think I'd prefer to take it out and manually add it back in while debugging a failing test.